client/components/domains/registrant-extra-info/docs/example.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+const countriesList = require( 'lib/countries-list' ).forDomainRegistrations(),
+	Card = require( 'components/card' ),
+	ExtraInfoFrForm = require( 'components/domains/registrant-extra-info/fr-form' ).default;
+
+class ExtraInfoFrFormExample extends PureComponent {
+	static displayName = 'DomainsFrExtraForm'
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
```

Comment:
Awesome, Thanks! I tried a static `initialState` with no joy, but didn't take that next logical step (nor go and read the manual) :p